### DISCUSSION TITLE
I have it installed, but I can't tell if it's working.

### DISCUSSION THREAD

**Discussion by ZukoIsBack** [Aelto/W3_RandomEncounters_Tweaks#190] [General]
I saw the message that it would roll out slowly, but I'm a couple hours in and haven't had an encounter. Is there a way to check? How long does it typically take to get going?

**Comment by Aelto** [comment]
[The page you get linked to at the end of the guide, or that is opened by the install script](https://aelto.github.io/tw3-random-encounters-reworked/rer-bible/tldr.html), answers both of your question!

**Reply by ZukoIsBack** [reply]
Thank you, I got the message, so I guess I just need to be patient. Drowners are still kicking my butt in enhanced edition, so that's not necessarily bad.
